package.json

```diff
@@ -1756,6 +1756,29 @@
               "onCommand:confluent.schemas.evolveSchemaSubject"
             ]
           },
+          {
+            "id": "cc-flink-statements",
+            "title": "Confluent Cloud for Apache Flink",
+            "description": "Author, submit, and view the result of Flink SQL statements in Confluent Cloud.\n[Enable Flink Support](command:confluent.support.openSettings)",
+            "media": {
+              "markdown": "resources/walkthrough/flink/statements.md"
+            },
+            "completionEvents": [
+              "onCommand:confluent.statements.create",
+              "onCommand:confluent.statements.refresh",
+              "onCommand:confluent.statements.viewstatementsql",
+              "onCommand:confluent.flinkStatementResults"
+            ]
+          },
+          {
+            "id": "copilot",
+            "title": "Copiloting Your Data Streaming",
+            "description": "The extension provides a Copilot-powered chat assistant for Confluent and Apache Kafka® developers, offering guidance on configuration, development best practices, and troubleshooting for Kafka clusters, Schema Registry, Flink, and streaming applications.",
```

Comment:
Should we add a note about setting controls for telemetry?

Reply:
Linking to the settings page via CTA button and the telemetry stuff is right there so I don't think we need to mention it here and keep it concise.